projects/tad/index.md

```diff
@@ -0,0 +1,84 @@
+
+The requirements are based on:
+
+- **ISO 25010 standard:** This standard defines the eight quality characteristics and provides a framework for evaluating software quality.
+- **Industry best practices:** This includes a broad range of recommendations and guidelines for IT tool development and implementation.
+- **Common IT tool requirements:** This information was gathered by analyzing various sources, such as documentation from popular IT tools, user reviews, and articles from reputable tech publications that discuss essential features and functionalities expected from different types of IT tools.
+- **Internal discussion and common sense**: While above sources are already exhaustive, we also used team discussions and our own knowledge.
+
+The requirements have been given a priority based on the [MoSCoW](https://en.wikipedia.org/wiki/MoSCoW_method) scale to allow for tool comparison.
+
+| ID   | Group                  | Requirement                                                  | Priority |
+| ---- | ---------------------- | ------------------------------------------------------------ | -------- |
+| 1... | Functionality          | The tool allows users to perform technical tests on an algorithm or model. | M        |
+|      | Functionality          | The tool allows users to choose which tests to perform.      | M        |
+|      | Functionality          | The tool supports plugin functionality so additional tests can be added easily. | S        |
+|      | Functionality          | The tool can generate a human readable report.               | M        |
+|      | Functionality          | The tool can generate a machine readable report.             | S        |
+|      | Functionality          | The tool allows various formats for reports, like PDF, JSON, MD. | C        |
+|      | Functionality          | The tool allows to create custom reports based on components. | S        |
+|      | Functionality          | It is possible to add custom components for reports.         | S        |
+|      | Functionality          | The tool provides logging.                                   | S        |
+|      | Functionality          | The tool can be used on an isolated system without an internet connection. | S        |
+|      | Functionality          | The tool does not share any data or logging to anyone or anywhere. | C        |
+|      | Functionality          | The tool can be integrated in a CI/CD flow.                  | C        |
+|      | Functionality          | The tool can be offered as a (cloud) service where no local installation is required. | C        |
+|      | Reliability            | The tool can operate consistently and reliably.              | M        |
+|      | Reliability            | The tool can recover automatically from common failures.     | M        |
+|      | Reliability            | The tool recovers from failures quickly and minimizes data loss. | S        |
```

Comment:
When does this fail? can you make it more specific?

Reply:
```suggestion
|      | Reliability            | This tool recovers from failures quickly, minimizing data loss, for example by automatically saving intermediate test progress results. | S        |
```